Libplanet/Net/Transports/NetMQTransport.cs

```diff
@@ -586,6 +593,8 @@ private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
                     );
                 }
             }
+
+            _logger.Verbose("ReceiveMessage ended. {@E}", e);
```

Comment:
How about using `nameof` keyword?

```suggestion
            _logger.Verbose($"{nameof(ReceiveMessage)} ended. {{@E}}", e);
```

Reply:
Same with discussion [above](https://github.com/planetarium/libplanet/pull/1242#discussion_r621902603).